### ISSUE TITLE
Convergence time

### ISSUE THREAD

**Issue by wangames** [PRIS-CV/PMG-Progressive-Multi-Granularity-Training#5]
Hello!
How many epoches does the training set iterate before it converges? In stanford cars dataset.

**Comment by RuoyiDu** [comment]
The results in our paper can be obtained with 200 epochs.

**Comment by RuoyiDu** [comment]
> Hello!
> How many epoches does the training set iterate before it converges? In stanford cars dataset.

The results in our paper can be obtained with 200 epochs.

**Comment by wangames** [comment]
Do you encounter the problem that loss does not decline when mutil-GPU  training.

**Comment by RuoyiDu** [comment]
> Do you encounter the problem that loss does not decline when mutil-GPU  training.

I've never encountered the similar problem. Make sure your torch version is correct.

**Comment by wangames** [comment]
I solve the problem by using pytorch1.3, thank you!
